examples/garrawarla/configs/config.yaml

```diff
@@ -0,0 +1,30 @@
+config:
+  build_stage:
+#    - $tempdir/$user/spack-stage
+    - $spack/var/spack/stage
+
+  # Directory in which to run tests and store test results.
+  # Tests will be stored in directories named by date/time and package
+  # name/hash.
+  test_stage: $spack/var/spack/test
+  
+  # Cache directory for already downloaded source tarballs and archived
+  # repositories. This can be purged with `spack clean --downloads`.
+  source_cache: $spack/var/spack/cache
+  
+  
+  # Cache directory for miscellaneous files, like the package index.
+  # This can be purged with `spack clean --misc-cache`
+  misc_cache: $spack/var/spack/misc-cache
+
+  # concretizer: `clingo` or `original` (good for debugging concretisations)
+  concretizer: clingo
+  #concretizer: original
+
+  install_tree:
+    root: /pawsey/mwa_sles12sp4/spack/software/
+
+  template_dirs:
+    - /astro/pawsey0001/maali/pawsey-spack-config/examples/garrawarla/templates
```

Comment:
maybe rename `maali` to `$USER`, as MarkO suggested they might just rename the `maali` user to `spack` everywhere